### DISCUSSION TITLE
Add Node-RED Node for Sending Payload to Raw WebSocket in Home Assistant

### DISCUSSION THREAD

**Discussion by klatka** [zachowj/node-red-contrib-home-assistant-websocket#1442] [Feature Request]
I would like to request the addition of a new node that should enable users to send custom payloads, such as updating the entity registry or any other configuration messages, directly through the WebSocket API of Home Assistant.

Purpose:
Home Assistant's WebSocket API offers powerful capabilities for interacting with the system in real-time. By adding a Node-RED node that can send raw payloads to this WebSocket, users can achieve more granular control and automation within their Home Assistant environments. This will be particularly useful for advanced users who need to perform specific tasks that are not covered by existing nodes.

Use Case:
A specific use case for this feature is updating the entity registry in Home Assistant. Currently, this requires a series of manual steps or custom scripts. With the proposed Node-RED node, users can automate this process. For example, to update an entity's ID, users can send the following payload:
```
{
    "id": index,
    "type": "config/entity_registry/update",
    "entity_id": entity_id,
    "new_entity_id": new_entity_id
}
```

**Comment by zachowj** [comment]
Check out the [API](https://zachowj.github.io/node-red-contrib-home-assistant-websocket/node/API.html) node.

It can send payloads to the API but can't subscribe to endpoints.

**Reply by klatka** [reply]
Works flawlessly, thx
